src/modules/flint/components/FlintAlerts/index.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import { Alert } from '../../../reusable'
+import { connect } from 'react-redux';
+import { UserIsFlintAffiliated } from '../../../flint'
+
+
+class FlintAlerts extends React.Component {  
+  getClosedStatus = () => {
+    return sessionStorage.getItem(`flint-alert-for-${this.props.datastore}-closed`) === "true"
```

Comment:
Considering using localStorage instead to save closed state across browser sessions. SessionStorage will be cleared when the page session ends.

What would users expect when they 'close' an alert?